### PR TITLE
All claims start controls mg

### PR DESCRIPTION
**Pull request by markgreenburg**
Adds standard form start controls to the intro page. This means:
- Users see different info alerts when they're logged in vs. logged out.
- Users are prompted to sign in via the modal if they hit one of the start buttons when they're logged out

Logged out:
screen shot 2018-08-09 at 11 04 08 am

Logged in:
screen shot 2018-08-09 at 11 04 45 am

Bottom is just a button (i.e., not info box):
screen shot 2018-08-09 at 1 58 44 pm

### CODE REVIEW

File: src/applications/disability-benefits/all-claims/config/form.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const formConfig = {
   intentToFileUrl: '/evss_claims/intent_to_file/compensation',
   submitUrl: `${environment.API_URL}/v0/disability_compensation_form/submit`,
   trackingPrefix: 'disability-526EZ-',
-  formId: '21-526EZ',
+  formId: '21-526EZ-all-claims',
```
Comment (cvalarida) [inline src/applications/disability-benefits/all-claims/config/form.js]: We should think about this; changing it later is...gonna be a pain, I think.

**Review by cvalarida — APPROVED** [review]
Had a few comments, but looks good on the whole.

File: src/applications/disability-benefits/all-claims/components/IntroductionPage.jsx [hunk 2]
```diff
@@ -128,15 +106,20 @@ class IntroductionPage extends React.Component {
             </li>
           </ol>
         </div>
-        <button className="temporary-start">Start the Disability Compensation Application</button>
+        <SaveInProgressIntro
+          prefillEnabled={this.props.route.formConfig.prefillEnabled}
+          formId={this.props.formId}
+          pageList={this.props.route.pageList}
+          startText="Start the Disability Compensation Application"
```
[the remark this answers is not in the document]
Comment (markgreenburg) [reply src/applications/disability-benefits/all-claims/components/IntroductionPage.jsx]: Ahh, good catch. I didn't realize this was a thing.

File: src/applications/disability-benefits/all-claims/components/IntroductionPage.jsx [hunk 3]
```diff
@@ -128,15 +106,20 @@ class IntroductionPage extends React.Component {
...
+          {...this.props.saveInProgress}/>
         <p>
           By clicking the button to start the disability application, you’ll declare your
           intent to file. This will reserve a potential effective date for when you could
           start getting benefits. You have 1 year from the day you submit your intent to
           file to complete your application.
         </p>
-        {/* TODO: Remove inline style after I figure out why .omb-info--container has a left padding */}
-        <div className="omb-info--container" style={{ paddingLeft: '0px' }}>
+        <div className="omb-info--container">
```
[the remark this answers is not in the document]
Comment (markgreenburg) [reply src/applications/disability-benefits/all-claims/components/IntroductionPage.jsx]: No left padding 😄

File: src/applications/disability-benefits/all-claims/config/form.js (hunk 1 above)
Comment (markgreenburg) [reply src/applications/disability-benefits/all-claims/config/form.js]: Yeah, I think I'd like to leave it as-is for now, but change it separately as soon as we can agree on a good id.

**Pull request merged**